### ISSUE TITLE
HeadlessException

### ISSUE THREAD

**Issue by olegs** [eseifert/vectorgraphics2d#40]
Hello vectorgraphics2d developers, thanks a lot for the great tool!
Recently I've tried to use version 0.10 to produce SVG graphics in headless enviroment and faced this exception:

```
Caused by: java.awt.HeadlessException: 
No X11 DISPLAY variable was set, but this program performed an operation which requires it.
        at sun.java2d.HeadlessGraphicsEnvironment.getDefaultScreenDevice(HeadlessGraphicsEnvironment.java:77)
        at de.erichseifert.vectorgraphics2d.VectorGraphics2D.<init>(SourceFile:113)
        at de.erichseifert.vectorgraphics2d.ProcessingPipeline.<init>(SourceFile:43)
        at de.erichseifert.vectorgraphics2d.SVGGraphics2D.<init>(SourceFile:46)
...
```
As far as I can see, the issue is already fixed in master, when can we expect the fix version published?

Thanks in advance,
Oleg

**Comment by seifertm** [comment]
Hi Oleg,

thanks for reaching out. You are right: VG2D should be able to work in headless mode, because the tests in TravisCI are also executed headless. Does the version in master work for you?

I think a release should be possible. I will talk to @eseifert about this.

**Comment by eseifert** [comment]
We will release VectorGraphics2D 0.11 today which will ship support for headless. I'm just trying to resolve an issue with upload to Maven Central. I'll close this issue, but feel free to reopen it if there are problems with headless support.

**Comment by olegs** [comment]
Thanks, a lot for the new release, this problem is fixed, I've submitted the new one: #42
